apps/sms/services/js/bridge_service_mixin.js

```diff
@@ -26,7 +26,7 @@
    * EVENTS, METHODS and STREAMS are all optional.
    */
   var serviceMixin = {
-    initService() {
+    initService(channel) {
```

Comment:
__recommendation:__ I'd rather name it ```endpoints``` and support array since service can listen on multiple endpoints, especially if shims hosted in single Window - want to talk to each other without using ```BroadcastChannel```. So we can have smth like this:
```
(endpoints || [self]).forEach((endpoint) => service.listen(endpoint));
```
or if we want to always listen for default endpoint:
```
// IIRC bridge internally uses Set to store endpoints, so we should not
// have duplicates if "endpoints" already included "self".
service.listen();

endpoints && endpoints.forEach((endpoint) => service.listen(endpoint));
```